### PR TITLE
Removed M14 10 rounders due to weight bugs, tweaked cigs, added appropriate stock game cosmetics and parachutes

### PR DESCRIPTION
**Pull request by focochlain**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -377,4 +377,5 @@ FodyWeavers.xsd
 # Additional files built by Visual Studio
 *.tlog
 
-# End of https://www.toptal.com/developers/gitignore/api/visualstudio
\ No newline at end of file
+# End of https://www.toptal.com/developers/gitignore/api/visualstudio
+docs/reference/tc_5-31_-_vc_boobytraps_text.pdf
```
Comment (wonkotron) [inline .gitignore:381]: What's this?

File: airgoons_mikeforce_customizations/config/arsenal.hpp [hunk 2]
```diff
@@ -401,8 +400,9 @@ class vn_whitelisted_arsenal_loadouts
 			{"vn_welrod_mag",{-1,0,-1,-1}},
 			{"murshun_cigs_matches",{0,0,0,0}},
 			{"vn_b_item_cigs_01", {0,0,0,0}},
-			{"murshun_cigs_cigpack", {2,2,2,2}},
+			{"murshun_cigs_cigpack", {2,0,2,2}},
 			{"immersion_cigs_cigar0_nv", {-1,-1,3,-1}},
+			{"immersion_cigs_cigar0", {-1,-1,3,-1}},
```
Comment (wonkotron) [inline airgoons_mikeforce_customizations/config/arsenal.hpp:405]: Should cigars be {-1,3,3,-1}?

File: .gitignore (hunk 1 above)
Comment (focochlain) [reply .gitignore:381]: A period US army manual on VC trap tactics I found, planning to use it as a reference for future iterations of the trap script

File: airgoons_mikeforce_customizations/config/arsenal.hpp (hunk 2 above)
Comment (focochlain) [reply airgoons_mikeforce_customizations/config/arsenal.hpp:405]: Normally yes but restrict by side is turned off in Mike Force so it just uses the lowest in any category.

File: .gitignore (hunk 1 above)
Comment (wonkotron) [reply .gitignore:381]: That's really cool.

File: airgoons_mikeforce_customizations/config/arsenal.hpp (hunk 2 above)
Comment (wonkotron) [reply airgoons_mikeforce_customizations/config/arsenal.hpp:405]: Good to know, thanks

**Pull request merged**